app/pages/order-task-list/helpers/generateTaskList.js

```diff
@@ -1,13 +1,27 @@
 import { baseUrl } from '../../../config';
 
-export const generateTaskList = ({ orderId, taskListManifest }) => {
+const convertSectionsDataToDict = ({ sectionsData }) => {
+  if (sectionsData && sectionsData.length > 0) {
+    const sectionsDataDict = sectionsData.reduce((dict, item) => (
+      { ...dict, [item.id]: item }), {});
+
+    return sectionsDataDict;
+  }
+  return undefined;
+};
+
+export const generateTaskList = ({ orderId, taskListManifest, sectionsData }) => {
+  const sectionsDataDict = convertSectionsDataToDict({ sectionsData });
+
   const taskLists = taskListManifest.tasks.map((task) => {
     const items = task.sections.map((section) => {
       const itemHref = section.enabled ? `${baseUrl}/organisation/${orderId}/${section.id}` : undefined;
+      const isItemComplete = sectionsDataDict && sectionsDataDict[section.id].status === 'complete' ? true : undefined;
```

Comment:
why can't you use false here?

Reply:
actually I know why - because of the table

Reply:
I don't know 😂 in BCC it was not provided so thought I had to do same. I'll try with false. Will make that line simpler